MSMQToAzureServiceBusFrame/App.config

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
```

Comment:
though .NetFramework 4.7.2 is still supported it would be best to use latest release - 4.8.1